app/src/services/api.ts

```diff
@@ -354,6 +355,15 @@ export const api = createApi({
       transformResponse: (response: { data: EmissionsFactorResponse }) =>
         response.data,
     }),
+    disconnectThirdPartyData: builder.mutation({
+      query: ({ inventoryId, inventoryValueId }) => ({
+        method: "DELETE",
+        url: `/inventory/${inventoryId}/inventoryvalue/${inventoryValueId}`,
```

Comment:
Could use the existing API endpoint that uses `value` instead of `inventoryvalue` and then passing in the SubCategory ID instead of the InventoryValue ID